man/git-brv.md

```diff
@@ -0,0 +1,31 @@
+git-brv(1) -- Show commit logs since some date
+========================================================
+
+## SYNOPSIS
+
+`git-brv` [&lt;date&gt;]
```

Comment:
It seems the `date` argument is not supported by the script?

Reply:
Good catch as well thanks

Reply:
fixed